services/repository-managers/src/main/java/org/sagebionetworks/repo/util/jrjc/CreatedIssue.java

```diff
@@ -0,0 +1,90 @@
+package org.sagebionetworks.repo.util.jrjc;
+
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+
+import java.util.Objects;
+
+public class CreatedIssue {
+
+    private String id;
+    private String key;
+    private String url;
+
+
+    public String getId() {
+        return id;
+    }
+
+    public void setId(String id) {
+        this.id = id;
+    }
+
+    public String getKey() {
+        return key;
+    }
+
+    public void setKey(String key) {
+        this.key = key;
+    }
+
+    public String getUrl() {
+        return url;
+    }
+
+    public void setUrl(String url) {
+        this.url = url;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o)
+            return true;
+        if (o == null)
+            return false;
+        if (getClass() != o.getClass())
+            return false;
+        CreatedIssue other = (CreatedIssue) o;
+        if (id == null) {
+            if (other.id != null)
+                return false;
+        } else if (!id.equals(other.id))
+            return false;
+        if (key == null) {
+            if (other.key != null)
+                return false;
+        } else if (!key.equals(other.key))
+            return false;
+        if (url == null) {
+            if (other.url != null)
+                return false;
+        } else if (!url.equals(other.url))
+            return false;
+        return true;
+    }
+
+    @Override
+    public int hashCode() {
+        final int prime = 31;
+        int result = 1;
+        result = prime * result + ((id == null) ? 0 : id.hashCode());
+        result = prime * result + ((key == null) ? 0 : key.hashCode());
+        result = prime * result + ((url == null) ? 0 : url.hashCode());
+        return result;
+    }
+
+    @Override
+    public String toString() {
+        return "CreatedIssue [id=" + id + ", key=" + key + ", url=" + url + "]";
+    }
+
+    public void initFromJSONObject(JSONObject jsonObject) {
```

Comment:
This should return a CreatedIssue and avoid overriding the instance (Maybe a static method or better an external mapper)